Sources/AppStoreConnectCLI/Services/Operations/ListUsersOperation.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+
+struct ListUsersOperation: APIOperation {
+
+    struct ListUsersDependencies {
+        let users: (APIEndpoint<UsersResponse>) -> Deferred<Future<UsersResponse, Error>>
```

Comment:
```suggestion
        let users: (APIEndpoint<UsersResponse>) -> AnyPublisher<UsersResponse, Error>
```
Generally you don't want to have specific publisher types exposed.

Reply:
Generally no, but `Deferred` and `Future` seem to the exception to this rule.
They give us a bit of information, that the requests will only be executed on subscription (`Deferred`) and that it will only return one result (`Future`).

It occurs to me that we don't really need `Deferred` in our instances. Getting results earlier is better. So I may simplify things just to be of type `Future`.